### PR TITLE
doc/revised overview

### PR DESCRIPTION
**Pull request by musslick**
# Description

I updated the main page of the documentation to reflect the current scope and goals of the AutoRA framework.

# Type of change
- **docs**: Documentation only changes

# Features (Optional)
- made overview figure transparent
- merged overview section with an introduction
- added additional sections, including features and usage

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,21 +1,72 @@
...
+- <b>Experimental design</b> for follow-up experiments
+- <b>Research documentation and dissemination</b>
+- <b>Closed-loop empirical research</b>
+- <b>Computational analyses of the scientific process</b> (metascience, computational philosophy of science)
 
 ## Motivation
 
-The integration of behavioral phenomena into computational models of human behavior is a fundamental staple of empirical research in the behavioral sciences, such as cognitive neuroscience and psychology. Yet, researchers are beginning to accumulate increasing amounts of data without having the temporal, computational, or monetary resources to integrate these data into scientific theories, and/or to test resulting theories in follow-up experiments. AutoRA seeks to accelerate the empirical research process, by incorporating existing machine learning techniques into a closed-loop system for the generation, estimation, and experimental validation of scientific models. In addition, AutoRA seeks to facilitate open science, by automating the documentation and dissemination of each step in the empirical research process. 
+The current state of various empirical sciences is beset by a replication crisis,
```
Comment (benwandrew) [inline docs/index.md:46]:
```suggestion
Various empirical sciences are beset by a replication crisis, 
```

File: docs/index.md [hunk 2]
```diff
@@ -1,21 +1,72 @@
...
+- <b>Closed-loop empirical research</b>
+- <b>Computational analyses of the scientific process</b> (metascience, computational philosophy of science)
 
 ## Motivation
 
-The integration of behavioral phenomena into computational models of human behavior is a fundamental staple of empirical research in the behavioral sciences, such as cognitive neuroscience and psychology. Yet, researchers are beginning to accumulate increasing amounts of data without having the temporal, computational, or monetary resources to integrate these data into scientific theories, and/or to test resulting theories in follow-up experiments. AutoRA seeks to accelerate the empirical research process, by incorporating existing machine learning techniques into a closed-loop system for the generation, estimation, and experimental validation of scientific models. In addition, AutoRA seeks to facilitate open science, by automating the documentation and dissemination of each step in the empirical research process. 
+The current state of various empirical sciences is beset by a replication crisis, 
+which can be attributed to inadequate precision of hypotheses, lack of transparency
```
Comment (benwandrew) [inline docs/index.md:47]:
```suggestion
which can be attributed to inadequately precise hypotheses, lack of transparency
```

File: docs/index.md [hunk 3]
```diff
@@ -1,21 +1,72 @@
...
+in research procedures, and insufficient rigor in testing findings. These limitations
+are the result of three primary bottlenecks, namely, a lack of formal modeling, the
```
Comment (benwandrew) [inline docs/index.md:49]:
```suggestion
are the result of three primary bottlenecks—a lack of formal modeling, the
```

File: docs/index.md [hunk 4]
```diff
@@ -1,21 +1,72 @@
...
+demanding requirements of open science, and a shortage of resources to reproduce 
+individual studies. Empirical scientists face difficulties in formalizing their 
+theories, find it arduous to document their research activities, and often lack 
+sufficient time and funds to conduct follow-up experiments to test and revise their
```
Comment (benwandrew) [inline docs/index.md:53]:
```suggestion
time and funds to conduct follow-up experiments to test and revise their
```

File: docs/index.md [hunk 5]
```diff
@@ -1,21 +1,72 @@
...
+hypotheses. These limitations serve as impediments to scientific progress and hinder
```
Comment (benwandrew) [inline docs/index.md:54]:
```suggestion
hypotheses. These limitations impede scientific progress and hinder
```

**Review by benwandrew — CHANGES_REQUESTED** [review]
suggestions for some alternative wording, all of which are of course optional. let's get merged and move onto the new documentation branch!

File: docs/index.md [hunk 6]
```diff
@@ -1,21 +1,72 @@
-# Autonomous Empirical Research
+# Automated Research Assistant
 
-AutoRA (Autonomous Research Assistant) is an open-source AI-based system for automating each aspect of empirical research in the behavioral sciences, from the construction of a scientific hypothesis to conducting novel experiments.
+<b>[AutoRA](https://pypi.org/project/autora/)</b> (<b>Au</b>tomated <b>R</b>esearch <b>A</b>ssistant) is an open-source framework for 
+automating multiple stages of the empirical research process, from the construction of a scientific
```
[the remark this answers is not in the document]
Comment (musslick) [reply docs/index.md]: Thanks, now addressed!

**Review by benwandrew — APPROVED** [review]
nice nice

**Pull request merged**